thundra/integrations/botocore.py

```diff
@@ -331,44 +301,42 @@ def __init__(self):
 
     def get_operation_name(self, wrapped, instance, args, kwargs):
         _, request_data = args
-        return request_data['Bucket'] if 'Bucket' in request_data else Constants.AWS_SERVICE_REQUEST
+        return request_data['Bucket'] if 'Bucket' in request_data else constants.AWS_SERVICE_REQUEST
 
     def getRequestType(self, string):
-        if string in Constants.S3RequestTypes:
-            return Constants.S3RequestTypes[string]
-        return Constants.AWS_SERVICE_REQUEST
-
-    def inject_span_info(self, scope, wrapped, instance, args, kwargs, response,
-                         exception):
+        if string in constants.S3RequestTypes:
+            return constants.S3RequestTypes[string]
+        return constants.AWS_SERVICE_REQUEST
```

Comment:
Return empty string instead of `AWS_SERVICE_REQUEST` constant if the bucket name is not available

Reply:
But be sure that in case of empty bucket name, operation name is not empty but `AWS_SERVICE_REQUEST` constant